.env.docker

```
@@ -103,3 +103,12 @@ AWS_PATH_STYLE=true
 
 FILESYSTEM_USER_FILES_ADAPTER_S3_BUCKET=test
 FILESYSTEM_USER_FILES_ADAPTER_S3_PREFIX=
+
+###> rostering file-upload ###
+FILESYSTEM_ROSTERING_FILES_ADAPTER_S3_BUCKET=
+FILESYSTEM_ROSTERING_FILES_ADAPTER_S3_PREFIX=
+S3_STORAGE_PENDING_FOLDER_NAME=pending
+S3_STORAGE_FAILED_FOLDER_NAME=failed
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The S3_STORAGE_FAILED_FOLDER_NAME key should go before the S3_STORAGE_PENDING_FOLDER_NAME key

(UnorderedKey)